modules/cluster-resources.adoc

```diff
@@ -0,0 +1,39 @@
+[id="support-cluster-resources_{context}"]
+= Interacting with your cluster resources
+
+You can interact with cluster resources using OpenShift CLI (`oc`) in {product-title}. The cluster resources that you see after running `oc api-resources` command are editable.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Access to the web console or the `oc` installed.
+
+.Procedure
+
+. To see what configuration operators applied, run the following command:
++
+[source,terminal]
+----
+$ oc api-resources -o name | grep config.openshift.io
+----
+
+. To know what cluster resources you can configure, run the following command:
++
+[source,terminal]
+----
+$ oc explain ${resource_name}.config.openshift.io
+----
+
+. To know configuration of CRD objects in the cluster, run the following command:
++
+[source,terminal]
+----
+$ oc get $config -o yaml
+----
+
+. To edit the cluster resource configuration, run the following commands:
++
+[source,terminal]
+----
+$ oc edit $config -o yaml
```

Comment:
Command here is not correct . should be  : `oc edit apiservers.config -o yaml`  .

Reply:
acked